src/scale_audio_rating.py

```diff
@@ -19,9 +19,17 @@
                                 PageBlobService)
 from azure.storage.file import FileService
 from jinja2 import Template
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
 
 import create_input as ca
 
+s = requests.Session()
+retries = Retry(total=5,
+            backoff_factor=0.1,
+            status_forcelist=[ 500, 502, 503, 504 ])
+
+s.mount('https://', HTTPAdapter(max_retries=retries))

```

Comment:
That's pretty interesting, I've typically done retries with just a try / except loop - I might have to give this a go